client/my-sites/stats/controller.jsx

```diff
@@ -176,6 +178,20 @@ export default {
 
 	site: function( context, next ) {
 		const { params: { site_id: givenSiteId }, query: queryOptions, store } = context;
+
+		if ( 'simplePaymentsEmailTour' === get( queryOptions, 'tour' ) ) {
+			if ( ! isDesktop() ) {
+				context.store.dispatch(
+					recordTracksEvent( 'calypso_simple_payment_email_tour', { mobile: true } )
```

Comment:
Perhaps `{ source: 'mobile' }` or `{ source: 'desktop' }`? A string value gives us more flexibility then a boolean if we ever want to track a third value.